<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Desktop</class>
 <widget class="QWidget" name="Desktop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>70</y>
     <width>331</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>Windows12 概念版</string>
   </property>
  </widget>
  <widget class="QLabel" name="task_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>550</y>
     <width>1031</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{background-color:rgb(255, 217, 101);}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>0</y>
     <width>1111</width>
     <height>561</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{background-color:rgb(197, 224, 179);}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="shutdown_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>F:/horiki/操作系统/课程设计/未标题-1.png</normaloff>F:/horiki/操作系统/课程设计/未标题-1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>31</width>
     <height>31</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="my_desktop">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>a.png</normaloff>a.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>90</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="task_manager">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>b.png</normaloff>b.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>90</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>label</zorder>
  <zorder>task_bar</zorder>
  <zorder>shutdown_btn</zorder>
  <zorder>my_desktop</zorder>
  <zorder>task_manager</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
